cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -1707,6 +1872,20 @@ SHOW INDEXES YIELD *
 ----
 SHOW CONSTRAINTS YIELD *
 ----
+
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CALL { RETURN 1 }
+----
+a|
+Unaliased expressions are deprecated in subquery `RETURN` clauses. Replaced by:
+[source, cypher, role="noheader"]
+----
+CALL { RETURN 1 AS one }
+----
```

Comment:
Was this cherry picked to `4.2` ???
Was this cherry picked to `4.3` ???
Was this cherry picked to `4.4` ???

Reply:
I backported it to 4.3 and 4.4, but not 4.2 as that is not supported anymore